packages/gatsby/src/utils/__tests__/cache.js

```diff
@@ -98,4 +105,36 @@ describe(`cache`, () => {
       containsThenMethod(cache.set(`a`, `b`))
     })
   })
+
+  describe(`set`, () => {
+    it(`resolves to the value it cached`, () => {
+      const cache = getCache()
+
+      return expect(cache.set(`a`, `b`)).resolves.toBe(`b`)
+    })
+
+    it(`rejects on caching error`, () => {
+      const cache = getCache()
+
+      mockErrorValue.mockImplementationOnce(() => !undefined)
```

Comment:
Could we use `mockReturnValueOnce`?

Reply:
Yes, we can. After further learning and playing with Jest, I tested with this exact suggestion and mockReturnValueOnce would be much cleaner in the case of both variables declaring both initially as just `jest.fn()`s.

Very new to this whole comment system and PRs. Should I just commit the change on my branch with the suggested change?